### DISCUSSION TITLE
help request: dose apisix suppots regex?

### DISCUSSION THREAD

**Discussion by rookie-fpc** [apache/apisix#8479] [Q&A]
### Description

I am a student who is learnning apisix. I want to know dose apisix suppots regex? And how can I use it if it can use regex.Thanks

### Environment

- APISIX version (run `apisix version`): 2.13
- Operating system (run `uname -a`): Centos7.9

**Comment by tzssangglass** [comment]
read more: https://github.com/apache/apisix/blob/master/docs/en/latest/FAQ.md#how-do-i-use-regular-expressions-regex-for-matching-uri-in-a-route
